content/security/advisories.html.haml

```diff
@@ -31,3 +31,17 @@ section: security
                   = page.title
                   - if page.kind
                     = "(#{page.kind})"
+                - if page.issues && year == '2018'
```

Comment:
`year[-2..-1].to_i >= 18` or `year.to_i >= 2018` instead in order to be valid even in 2019+ ?